docs/tutorial/part-eight/index.md

```diff
@@ -205,7 +205,7 @@ module.exports = {
 }
 ```
 
-3. In the `src/components` directory, create a file called `seo.js` and add the following:
+3.  Erstelle eine Datei namens `seo.js` im Ordner `src/components` und schreibe folgendes in die Datei:
```

Comment:
```suggestion
3.  Erstelle eine Datei namens `seo.js` im Ordner `src/components` mit folgenden Inhalt:
```

It sounds a bit childish to me, what do you think?

Reply:
True. How would you like the following?
> Erstelle eine Datei namens `seo.js` mit folgendem Inhalt im Ordner `src/components`: